CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CanHazFunny;
+
+    public class Jester
+    {
+        public IJokeServiceInterface? JokeService { get; set; }
+        public IPrintJokeInterface? Printer { get; set; }
+
+        public Jester(IJokeServiceInterface? jokeService, IPrintJokeInterface? printer)
+        {
+            this.JokeService = jokeService;
+            this.Printer = printer;
```

Comment:
```suggestion
            JokeService = jokeService;
            Printer = printer;
```
`this` is not needed